.gitignore

```diff
@@ -3,6 +3,7 @@
 
 # Jetbrains - WebStorm IDE files
 .idea
+.DS_Store
```

Comment:
`.DS_Store` is actually a MacOS metadata folder.

Could you move it under its own comment?
# MacOS

Reply:
+1